### PR TITLE
Resizable chartboxes [12029]

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

**Review by jparisu — CHANGES_REQUESTED** [review]
Error in hide Shortcut bar

File: qml/MonitorMenuBar.qml [hunk 1]
```diff
@@ -72,6 +76,95 @@ MenuBar {
...
+                    text: menuItem.text
+                    opacity: enabled ? 1.0 : 0.3
+                    color: menuItem.checked ? Theme.eProsimaLightBlue : Theme.grey
+                    horizontalAlignment: Text.AlignLeft
+                    verticalAlignment: Text.AlignVCenter
+                    elide: Text.ElideRight
+                }
+            }
+        }
+        MenuSeparator { }
```
[the remark this answers is not in the document]
Comment (rsanchez15) [reply qml/MonitorMenuBar.qml:167]: Addressed in 21b5d1e

File: qml/CustomLegend.qml [hunk 2]
```diff
@@ -21,6 +21,7 @@ import QtQml.Models 2.15
 
 Rectangle {
     id: legend
+    implicitHeight: itemHeight * gridView.count
```
[the remark this answers is not in the document]
Comment (rsanchez15) [reply qml/CustomLegend.qml]: Addressed in e68feba

File: qml/MonitorMenuBar.qml [hunk 3]
```diff
@@ -72,6 +76,95 @@ MenuBar {
...
+                        height: 26
+                        anchors.centerIn: parent
+                        visible: menuItem.checkable
+                        IconSVG {
+                            id: participantIcon
+                            name: menuItem.iconName
+                            size: parent.width
+                            anchors.centerIn: parent
+                            visible: menuItem.checkable
+                            color: menuItem.checked ? "eProsimaLightBlue" : "grey"
```
[the remark this answers is not in the document]
Comment (rsanchez15) [reply qml/MonitorMenuBar.qml]: Addressed in 5cc221f

**Review by jparisu — APPROVED** [review]
LGTM
Waiting for CI

**Pull request merged**